src/ts/test/helpers/fixtures.ts

```diff
@@ -50,42 +50,69 @@ export const createBlock = (
   id: `0-from:${from}-to:${to}`
 });
 
+export interface ICreateMatcherResponseSpec {
+  from: number;
+  to: number;
+  block?: IBlock;
+  wordFrom?: number;
+  wordTo?: number;
+  category?: ICategory;
+  suggestions?: ISuggestion[];
+}
+
 export const createMatcherResponse = (
-  from: number,
-  to: number,
-  wordFrom: number = from,
-  wordTo: number = from + 3,
-  category = {
-    id: "1",
-    name: "Cat",
-    colour: "eeeee"
-  },
-  suggestions = [] as ISuggestion[],
-  requestId = exampleRequestId
-): IMatcherResponse => ({
-  requestId,
-  categoryIds: [category.id],
-  blocks: [
-    {
-      id: createBlockId(0, from, to),
-      from,
-      to,
-      text: "block text"
-    }
-  ],
-  matches: [
+  specs: ICreateMatcherResponseSpec[],
```

Comment:
I had to add a bit of detail to the fixture here to make it play nicely with the plugin state – we're testing more of it now, so we need a more accurate representation of what we'd get from a response.